src/amp/components/elements/YoutubeBlockComponent.tsx

```diff
@@ -51,8 +51,6 @@ export const YoutubeBlockComponent: React.FC<{
 	if (element.channelId) {
```

Comment:
question whether the related videos data attribute still needs to be inside this if statement, but thought i'd change only what was necessary....